ext/maxminddb.c

```diff
@@ -405,11 +467,29 @@ static zend_class_entry *lookup_class(const char *name TSRMLS_DC)
         zend_error(E_ERROR, "Class %s not found", name);
     }
     return *ce;
+#endif
 }
 
+#ifdef ZEND_ENGINE_3
+static void maxminddb_free_storage(zend_object *object TSRMLS_DC)
+{
+    maxminddb_obj *obj = php_maxminddb_fetch_object(object TSRMLS_CC);
+    if (obj->mmdb != NULL) {
+        MMDB_close(obj->mmdb);
+        efree(obj->mmdb);
+    }
+
+    if (obj->std.properties != NULL) {
+        zend_hash_destroy(obj->std.properties);
+        FREE_HASHTABLE(obj->std.properties);
+    }
+
+    //efree(obj);
```

Comment:
I'm not exactly sure, it may have to do with the refcounting changes. It seems to be standard in extensions though:

https://github.com/php/php-src/blob/php-5.6.5/ext/mysqli/mysqli.c#L214
https://github.com/php/php-src/blob/master/ext/mysqli/mysqli.c#L205

https://github.com/php/php-src/blob/php-5.6.5/ext/dom/php_dom.c#L1088
https://github.com/php/php-src/blob/master/ext/dom/php_dom.c#L1026